File: CS/HTML_Export_ImageSourceExample/CustomUriProvider.cs
#region #usings
using DevExpress.Office.Services;
using DevExpress.Office.Utils;
using System;
#endregion #usings

namespace Retain_Img_Src {
    #region #customuriprovider
    public class CustomUriProvider : IUriProvider {
        public string CreateCssUri(string rootUri, string styleText, string relativeUri) {
            return String.Empty;
        }

        public string CreateImageUri(string rootUri, OfficeImage image, string relativeUri) {
            return image.Uri;
        }
    }
    #endregion #customuriprovider
}

File: VB/HTML_Export_ImageSourceExample/CustomUriProvider.vb
Imports DevExpress.Office.Services
Imports DevExpress.Office.Utils
Imports System


Namespace Retain_Img_Src
	Public Class CustomUriProvider
		Implements IUriProvider

		Public Function CreateCssUri(ByVal rootUri As String, ByVal styleText As String, ByVal relativeUri As String) As String Implements IUriProvider.CreateCssUri
			Return String.Empty
		End Function

		Public Function CreateImageUri(ByVal rootUri As String, ByVal image As OfficeImage, ByVal relativeUri As String) As String Implements IUriProvider.CreateImageUri
			Return image.Uri
		End Function
	End Class
End Namespace